### PR TITLE
docs(provider/kubernetes): Add doc support for strategy.spinnaker.io/…

### PR DESCRIPTION
**Pull request by gojenkuser**
…use-source-capacity reserved annotation.

### CODE REVIEW

File: reference/providers/kubernetes-v2/index.md [hunk 1]
```diff
@@ -128,6 +128,14 @@ command](https://kubernetes.io/docs/reference/generated/kubectl/kubectl-commands
   ConfigMap or Secret to be deployed without appending a new version when the
   contents change, for example.
 
+* `strategy.spinnaker.io/use-source-capacity`
+
+  When set to `'true'` or `'false'`, this overrides the resources replicas size
```
Comment (dorbin) [inline reference/providers/kubernetes-v2/index.md]: s/resources/resource's/
and
s/replicas/replica/  
?

Or should it be "...overrides the resource replica's size..."?

File: reference/providers/kubernetes-v2/index.md [hunk 2]
```diff
@@ -128,6 +128,14 @@ command](https://kubernetes.io/docs/reference/generated/kubectl/kubectl-commands
...
+  with the currently deployed resource size. This is supported for 
+  deployment, replicaset or stateful set.
```
Comment (dorbin) [inline reference/providers/kubernetes-v2/index.md]: s/replicaset/ReplicaSet/

**Review by dorbin — DISMISSED** [review]

File: reference/providers/kubernetes-v2/index.md (hunk 1 above)
Comment (lwander) [reply reference/providers/kubernetes-v2/index.md]: I think it should be
"this overrides the resource's replica count"

File: reference/providers/kubernetes-v2/index.md [hunk 3]
```diff
@@ -128,6 +128,14 @@ command](https://kubernetes.io/docs/reference/generated/kubectl/kubectl-commands
   ConfigMap or Secret to be deployed without appending a new version when the
   contents change, for example.
 
+* `strategy.spinnaker.io/use-source-capacity`
+
+  When set to `'true'` or `'false'`, this overrides the resources replicas size
+  with the currently deployed resource size. This is supported for 
+  deployment, replicaset or stateful set. 
+  This can be used to allow resizing a resource and maintaining the same
```
Comment (lwander) [inline reference/providers/kubernetes-v2/index.md]: I would reword to ".. to allow resizing a resource in the Spinnaker UI or with `kubectl` without overriding the new size during subsequent manifest deployments" to make it clear where the resizing can happen.

**Review by dorbin — COMMENTED** [review]
LGTM

@lwander ?

**Review by lwander — APPROVED** [review]
